packages/flutter/lib/src/rendering/error.dart

```diff
@@ -50,6 +51,9 @@ class RenderErrorBox extends RenderBox {
 
   /// The message to attempt to display at paint time.
   final String message;
+  final double systemTopPadding = Zone.current['systemTopPadding'] == null
```

Comment:
In testing, there's no way to control the values you get back from the `window` object.  For example, @xster wants to test how his render object behaves with different `window.padding.top` values.  Grabbing the window from the Zone lets you run your test in a Zone where you've substituted a different implementation of Window for `window`.